python/lsst/summit/extras/focusAnalysis.py

```diff
@@ -476,7 +476,7 @@ def getFocusData(self, dayObs, seqNums, *, manualCentroid=None, doCheckDispersed
             exp = self._bestEffort.getExposure(dataId)
 
             # sanity/consistency checking
-            filt = exp.getFilterLabel().physicalLabel
+            filt = exp.filterLabel.physicalLabel
```

Comment:
Has this been checked to work?  Pulling up an exposure in my open python window tells me that `ExposureF has no attribute 'filterLabel'`.  I see a `filter` attribute, but no filterLabel.  It looks like a recent change, so if you're not on main `afw` you might not see it.
https://github.com/lsst/afw/commit/ae33f7b6e9a0437c374e81655991ba9bdfd3feb8

Reply:
Totally right, here and throughout the other review, thank you!